Spigot-Server-Patches/0019-Implement-Paper-VersionChecker.patch

```diff
@@ -84,11 +84,12 @@ index 0000000000000000000000000000000000000000..c8b911e5d013525ffc5d2911ee0e421d
 +        if (siteApiVersion == null) { return -1; }
 +        try {
 +            try (BufferedReader reader = Resources.asCharSource(
-+                new URL("https://papermc.io/api/v1/paper/" + siteApiVersion + "/latest"),
++                new URL("https://papermc.io/api/v2/projects/paper/versions/" + siteApiVersion),
 +                Charsets.UTF_8
 +            ).openBufferedStream()) {
 +                JsonObject json = new Gson().fromJson(reader, JsonObject.class);
-+                int latest = json.get("build").getAsInt();
++                JsonArray builds = json.getAsJsonArray("builds");
++                int latest = builds.get(builds.size()-1).getAsInt();
```

Comment:
Feels appropriate to just use a stream here to find max to be 100% sure...

Reply:
Added more streams! :)